### DISCUSSION TITLE
Resistor R2 ?

### DISCUSSION THREAD

**Discussion by vbugarsk** [volw/wifiOiler#4] [Q&A] [answered]
Hi @volw, I am trying to make motorcycle chain oiler by taking some parts from your schematic, oiler from Rehtronik, and oiler from McCoi (www.mccoi.de). It won't be based on GPS to calculate distance, but rather on speed signal from motorcycle (Hall sensor).

I am curious about **resistor R2** in your schematic (one on the net going from MCU D7 port to the Mosfet gate) **Is it really needed ?** I didn't see pull-down resistor on Rehtronik, neither on McCoi oilers,

By the way, you did great job with this oiler, super tidy PCB, and great software design :)

**Comment by volw** [comment, marked as answer]
Hi buggy09,

just found your mail… 

R2 is to ensure that the pump is’nt triggered when the oiler is switched on (because of undefined power-states of pin)  - has always happened before. 

And thanks for the nice feedback!
